src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -45,7 +45,7 @@ const ChannelTitle = ({
       marginRight: 4
     },
     chatContentUnreadCountContainer: {
-      backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue,
+      backgroundColor: isAnonymousTab ? colors.anon_primary : colors.darkBlue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors.bondi_blue` to `colors.anon_primary` seems fine as long as the new color `anon_primary` is defined in the `colors` module and it meets the design requirements for the anonymous tab. Ensure that this color change doesn't affect the readability or user experience.
```diff
-      backgroundColor: isAnonymousTab ? colors.bondi_blue : colors.darkBlue,
+      backgroundColor: isAnonymousTab ? colors.anon_primary : colors.darkBlue,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->